lib/main.js

```diff
@@ -5,10 +5,25 @@ var parse = require("./parse.js");
 var filters = {};
 var Lexer = parse.Lexer;
 var Parser = parse.Parser;
+var parserOptions = {
+    csp: false, // noUnsafeEval,
```

Comment:
https://docs.angularjs.org/api/ng/directive/ngCsp

I have the same idea, I don't want eval so false (right?)